cmds/tui.c

```diff
@@ -212,6 +214,12 @@ static char *selected_report_sort_key[NUM_REPORT_FIELD];
 
 static int curr_sort_key = 0;
 
+static void check_utf8_locale(void)
+{
+	const char *lang = getenv("LANG");
+	is_utf8_locale = (lang != NULL && strstr(lang, ".UTF-8") != NULL);
```

Comment:
I worry if checking `.UTF-8` from `LANG` is enough because the following variations are also possible.
```
"utf-8", "utf8", "utf_8", "utf.8", "UTF-8", "UTF8", "UTF_8", "UTF.8"
```

Reply:
I wonder if there's any distro actually uses such a variation.

Reply:
I manually used `utf8` myself, but not sure where I found and set it that way. Maybe someone might do.

Reply:
I think it's ok to check ".UTF-8" as it seems the common case.

Reply:
Then I'm fine with that.